Subject: fix Makefile.am to not install examples as normal documentation
Author: Jan Dittberner <jandd@debian.org>
Description: upstream's Makefile.am defines variables to install the example
 lua scripts as documentation. The Debian package installs these lua scripts as
 examples so the upstream macros are removed by this patch
--- a/Makefile.am
+++ b/Makefile.am
@@ -12,15 +12,6 @@
 endif
 
 lsyncd_LDADD = $(LUA_LIBS)
-exampledir = $(docdir)/
-dist_example_DATA = \
-	examples/lbash.lua \
-	examples/lecho.lua \
-	examples/lgforce.lua \
-	examples/limagemagic.lua \
-	examples/lpostcmd.lua \
-	examples/lrsync.lua \
-	examples/lrsyncssh.lua
 TESTS = \
 	tests/churn-rsync.lua \
 	tests/churn-rsyncssh.lua \
